sources/packages/backend/apps/api/src/services/institution/institution.service.ts

```diff
@@ -694,6 +694,27 @@ export class InstitutionService extends RecordDataModelService<Institution> {
       .getOne();
   }
 
+  /**
+   * Get the institutionType by institution id.
+   * @param institutionId Institution id.
+   * @returns Institution retrieved, if found, otherwise returns null.
+   */
+  async getInstitutionTypeById(institutionId: number): Promise<Institution> {
+    return this.repo.findOne({
```

Comment:
if the where is just an id, we can use findonebyId instead of findOne, just a suggestion

Reply:
In that case, I will leave it as it is.

Reply:
The findOneBy here wouldn't bring the institutionType. AFAIK findOneBy doesn't allow you to specify the relations.